metron-deployment/vagrant/full-dev-platform/README.md

```diff
@@ -24,35 +26,38 @@ Any platform that supports these tools is suitable, but the following instructio
 
 1. Install Homebrew by following the instructions at [Homebrew](http://brew.sh/).
 
-2. Run the following command in a terminal to install all of the required tools.
+1. Run the following command in a terminal to install all of the required tools.
 
-  ```  
-  brew cask install vagrant virtualbox java
-  brew install maven git
-  ```
+    ```  
+    brew cask install vagrant virtualbox java docker
+    brew install maven git
+    ```
 
-3. Install Ansible by following the instructions [here](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip).
+1. Install Ansible by following the instructions [here](http://docs.ansible.com/ansible/intro_installation.html#latest-releases-via-pip).
 
 ### Deploy Metron
 
-1. Install Vagrant Hostmanager.
+1. Build Metron
 
-  ```
-  vagrant plugin install vagrant-hostmanager
-  ```
+    ```
+    cd incubator-metron
+    mvn clean install -DskipTests
+    ```
 
```

Comment:
Leaving the build step in quick dev must have been my error. It doesn't belong there. I think good omitting them both. Looks like 538 is on the way to closure.